### DISCUSSION TITLE
Site só abre corretamente se atualizar.

### DISCUSSION THREAD

**Discussion by Nolla86** [frontendbr/forum#2240] [Perguntas e Respostas]
olá a todos, estou estudando/editando um site e o mesmo está dando erro quando o acesso pela primeira vez ou de vez em quando. 

O menu fica com as imagens pequenas e buga uns efeitos no scroll, porém quando atualizo (F5), tudo sempre funciona corretamente.

o site: https://webtestesrn.com/testes/testes2/

Detalhe que quando acesso offline/raiz daqui do computador ele abre normalmente.

Podem me ajudar? O que eu faço?

Agradeço a todos desde já.

Felipe Nolla

**Comment by gabriel-hahn** [comment]
Fala Felipe! Aqui abriu o site normalmente, você já identificou e corrigiu o problema? Aliás, muito legal o site e uma pena esse studio de tattoo ser tão longe de onde eu moro, gostei haha

**Comment by igorthurow** [comment]
Fala Felipe!

Aqui também ta abrindo bacana. Site tá massa bro, parabéns. Não é tão simples pensar ou desenvolver fora da caixa. Imagino que deve ter sido gostosinho implementar esse layout haha.

Abraço!
